docs/docusaurus/docs/guides/setup/setup_overview.md

```diff
@@ -10,79 +10,62 @@ import UniversalMap from '@site/docs/images/universal_map/_universal_map.mdx';
 
 <UniversalMap setup='active' connect='inactive' create='inactive' validate='inactive'/>
 
-<!-- Only keep one of the 'To best understand this document' lines.  For processes like the Universal Map steps, use the first one.  For processes like the Architecture Reviews, use the second one. -->
+Setting up Great Expectations includes installing Great Expectations and initializing your deployment. Optionally, you can customize the configuration of some components, such as Stores, Data Docs, and Plugins.
 
-Getting started with Great Expectations is quick and easy.  Once you have completed setup for your production deployment, you will have access to all the features of Great Expectations from a single entry point: Your <TechnicalTag relative="../" tag="data_context" text="Data Context" />.  You will also have your <TechnicalTag relative="../" tag="store" text="Stores" /> and <TechnicalTag relative="../" tag="data_docs" text="Data Docs" /> configured in the manner most suitable for your project's purposes.
+After you've completed the setup for your production deployment, you can access all Great Expectations features from your <TechnicalTag relative="../" tag="data_context" text="Data Context" />.  Also, your <TechnicalTag relative="../" tag="store" text="Stores" /> and <TechnicalTag relative="../" tag="data_docs" text="Data Docs" /> will be optimized for your business requirements.
 
-If you're not interested in managing your own configuration or infrastructure, then Great Expectations Cloud may be of interest to you. You can learn more about Great Expectations Cloud — our fully managed SaaS offering — by signing up for [our weekly cloud workshop](https://greatexpectations.io/cloud). You’ll get to preview our newest features and apply for our private Alpha program!
+To set up <TechnicalTag relative="../" tag="datasource" text="Datasources" />, <TechnicalTag relative="../" tag="expectation_suite" text="Expectation Suites" />, and <TechnicalTag relative="../" tag="checkpoint" text="Checkpoints" /> see the specific topics for these components. 
+
+If you don't want to manage your own configurations and infrastructure, then Great Expectations Cloud might be the solution. To learn more about Great Expectations Cloud, sign up for the [weekly cloud workshop](https://greatexpectations.io/cloud). If you're interested in participating in the Great Expectations Cloud Beta program, or you want to receive progress updates, [**sign up for the Beta program**](https://greatexpectations.io/cloud).
```

Comment:
I'd double check on the wording of this section -- mostly because I think it's one of the earlier instances where we mentioned GX Cloud, and I'm not certain if the language around it ("Beta program" might just be "Cloud Beta", now, and "weekly cloud workshop" might be "weekly GX Cloud workshop") or something like that.  Also, I'm not certain if the weekly cloud workshop has changed to some other onboarding method.

Reply:
@Rachel-Reverie I removed the sentence. The text now better matches our other Cloud admonition statements.